cmd/botkube/main.go

```diff
@@ -207,7 +208,7 @@ func newLogger(logLevelStr string) *logrus.Logger {
 		logLevel = logrus.InfoLevel
 	}
 	logger.SetLevel(logLevel)
-	logger.Formatter = &logrus.TextFormatter{ForceColors: true, FullTimestamp: true}
+	logger.Formatter = &logrus.TextFormatter{FullTimestamp: true, DisableColors: logDisableColors}
```

Comment:
What is the default value of `ForceColors`? In case if `logDisableColors=false` do we need to set `ForceColors: true`?

Reply:
Default for `ForceColors` is `false`. I'm not sure if we need to set `ForceColors` at all, because as the comment says: "Set to true to bypass checking for a TTY before outputting colors.". I think we shouldn't bypass the check 🤔 
![Screenshot 2022-06-07 at 14 02 31](https://user-images.githubusercontent.com/7155799/172374348-aa51a541-557e-4507-9976-e1a9efc6640a.png)